recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set version = "0.15.0" %}
+{% set version = "0.16.0" %}
```

Comment:
Do we have any objections to building the latest version 0.17.0?
I do not see breaking changes https://github.com/apache/thrift/blob/v0.17.0/CHANGES.md#0170
I'm not sure if we should update `thrift-cpp` too.